src/lib/dnssd/Resolver_ImplMinimalMdns.cpp

```diff
@@ -397,11 +420,11 @@ void MinMdnsResolver::AdvancePendingResolverStates()
             }
 
             mActiveResolves.Complete(nodeData.operationalData.peerId);
-            if (mOperationalDelegate != nullptr)
+            for (auto & delgate : mOperationalDelegates)
             {
-                mOperationalDelegate->OnOperationalNodeResolved(nodeData);
+                delgate->OnOperationalNodeResolved(nodeData);
```

Comment:
How does this play along with our existing resolve-cancellation machinery and whatnot?

Reply:
added a new delegate in stead